extensions/common/vault/vault-aws/build.gradle.kts

```diff
@@ -18,17 +18,7 @@ plugins {
 
 dependencies {
     api(project(":spi:common:core-spi"))
-    implementation("software.amazon.awssdk:secretsmanager:2.19.3")
+    api(libs.aws.secretsmanager)
```

Comment:
```suggestion
    implementation(libs.aws.secretsmanager)
```

Reply:
Thanks. Didn't know the difference between api and implementation. The more you know...